src/components/Header.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Login from './Login.jsx';
+
+/**
+ * @function ErrorPage - Error page Component.
+ * @return {string} - HTML that displays when user is accessing
```

Comment:
The comments are confusing. can you do away with comments if it makes the code look ambigous?